xbmc/addons/AddonManager.h

```diff
@@ -117,6 +117,8 @@ namespace ADDON
 
     bool GetInstallableAddons(VECADDONS& addons, const TYPE &type);
 
+    bool GetInstalledBinaryAddonPaths(std::vector<std::tuple<std::string, bool, std::string>>& binaryAddonList);
```

Comment:
+ please add documentation, esp. the content of the vector is far from being self-explaining. ;-)
+ you should introduce a typedef for the complex type used here:
<code>typedef std::vector<std::tuple<std::string, bool, std::string>> BinaryAddonList;</code>